src/main/java/io/javalin/plugin/openapi/OpenApiHandler.kt

```diff
@@ -13,11 +13,27 @@ class OpenApiHandler(app: Javalin, val options: OpenApiOptions) : Handler {
     private val handlerMetaInfoList = mutableListOf<HandlerMetaInfo>()
 
     init {
-        app.events { it.handlerAdded { handlerInfo -> handlerMetaInfoList.add(handlerInfo) } }
+        app.events { it.handlerAdded { handlerInfo ->
+            if(handlerInfo.httpMethod.isHttpMethod()){
+                handlerMetaInfoList.add(handlerInfo)
+            }
+        } }
     }
 
     fun createOpenAPISchema(): OpenAPI = JavalinOpenApi.createSchema(CreateSchemaOptions(
-            handlerMetaInfoList = handlerMetaInfoList,
+            handlerMetaInfoList = handlerMetaInfoList.filter { handler ->
```

Comment:
This seems unnecessarily convoluted. There is also a convention in Javalin for exact vs prefix:


`/exact-path` and `/prefix-path/*`

We should probably stick to the same setup here?

Reply:
I don't really get what you meant here, pls can you explain it a little more detailed?

Reply:
Sure. Currently, this is the API: 

```kotlin
options.ignoreDocumentation("/my-endpoint", HttpMethod.GET, HttpMethod.POST)
options.ignoreDocumentationWithPathPrefix("/my-subpath", HttpMethod.GET, HttpMethod.POST)
```

Instead it could be:

```kotlin
options.ignoreDocumentation("/my-endpoint", HttpMethod.GET, HttpMethod.POST)
options.ignoreDocumentation("/my-subpath/*", HttpMethod.GET, HttpMethod.POST)
```

This would be more in line with how the other APIs in Javalin work. Example from docs (https://javalin.io/documentation#before-handlers): 

```kotlin
app.before("/path/*") { ctx ->
    // runs before request to /path/*
}
```

On an unrelated note, I think `ignoreHandler` or `ignoreRoute` is a better name than `ignoreDocumentation` 🤔

Reply:
This is way more fluent, sure i will implement it :)